<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>531</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::TabFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
   padding: 5px;
}
QPushButton:focus {
    /* border: 1px dotted black; */
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_1">
         <attribute name="title">
          <string>Login</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QPushButton" name="btn_login">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Log in</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_logout">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Log out</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_check_status">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Check login status</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Minimum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btn_quit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Quit</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Empire Status</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QPushButton" name="btn_get_empire_status">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Get Empire Status</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="txt_status">
              <property name="focusPolicy">
               <enum>Qt::ClickFocus</enum>
              </property>
              <property name="tabChangesFocus">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>Abbreviations</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QTableWidget" name="tbl_abbrv">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="sortingEnabled">
               <bool>true</bool>
              </property>
              <property name="columnCount">
               <number>2</number>
              </property>
              <column/>
              <column/>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>451</width>
     <height>25</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
    <addaction name="separator"/>
    <addaction name="actionChose_Config_File"/>
    <addaction name="actionChose_Config_Section"/>
    <addaction name="actionConfig_File_Status"/>
    <addaction name="separator"/>
    <addaction name="actionLog_In"/>
    <addaction name="separator"/>
    <addaction name="actionTest"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>status tt</string>
   </property>
   <property name="statusTip">
    <string>status st</string>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionChose_Config_File">
   <property name="text">
    <string>Chose Config File</string>
   </property>
  </action>
  <action name="actionChose_Config_Section">
   <property name="text">
    <string>Chose Config Section</string>
   </property>
  </action>
  <action name="actionTest">
   <property name="text">
    <string>Test</string>
   </property>
  </action>
  <action name="actionLog_In">
   <property name="text">
    <string>Log In</string>
   </property>
  </action>
  <action name="actionConfig_File_Status">
   <property name="text">
    <string>Config File Status</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>btn_login</tabstop>
  <tabstop>btn_logout</tabstop>
  <tabstop>btn_check_status</tabstop>
  <tabstop>btn_quit</tabstop>
  <tabstop>btn_get_empire_status</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_quit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MainWindow</sender>
   <signal>destroyed()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
